backend-api/src/functions/asyncActiveSession/tokenService/tokenService.ts

```diff
@@ -6,43 +6,39 @@ import {
 import { errorResult, Result, successResult } from "../../utils/result";
 import { KMSAdapter } from "../../adapters/kmsAdapter";
 import crypto from "crypto";
+import { jwtVerify, JWTVerifyResult, KeyLike } from "jose";
+import { IJwks, IPublicKeyGetter } from "./publicKeyGetter";
 
```

Comment:
```suggestion
type TokenDependencies = {
   publicKeyGetter: IPublicKeyGetter;
 };

 const tokenDependencies: TokenDependencies = {
   publicKeyGetter: new PublicKeyGetter(),
 };
```

Reply:
This way we don't need this logic in the handler dependencies and we can instantiate PublicKeyGetter in this file?

Reply:
Resolving in the next PR